kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -0,0 +1,260 @@
+{
+    "administration": {
+        "category": "administration",
+        "tags": [
+            "association_or_organization",
+            "city_hall",
+            "immigration_and_naturalization_service"
+        ]
+    },
+    "airport": {
+        "category": "airport",
+        "tags": [
+            "international_airport",
+            "airport"
+        ]
+    },
+    "artCulture": {
+        "category": "artCulture",
```

Comment:
done, snake case. I am also more comfortable with snake case. :)

Reply:
The snake case should only be in the category property. The JSON key should be camel case since that is the JSON naming convention. So for example:

```json
{
  "artCulture": {
    "category": "art_culture",
    "tags": […]
  }
}
```